test-case/verify-kernel-module-load-probe.sh

```diff
@@ -13,15 +13,18 @@ set -e
 ##    find sof relative module
 
 # source from the relative path of current folder
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
 
 func_opt_parse_option "$@"
 
 setup_kernel_check_point
 
 dlogi "Checking if sof relative modules loaded"
 dlogc "lsmod | grep \"sof\""
-lsmod | grep "sof"
-[[ $? -ne 0 ]] && dloge "No available sof module found! Dumping lsmod:" && lsmod && exit 1
 
-exit 0
+lsmod | grep "sof" || {
+    dloge "No available sof module found! Dumping lsmod:"
+    lsmod
+    exit 1
```

Comment:
```suggestion
lsmod | grep "sof" || {
    lsmod 
    die "No available sof module found!
}
```
(If you can't recognize the output of lsmod, you have no business using this test)

Reply:
Done.